go/client/cmd_simplefs.go

```diff
@@ -39,16 +38,8 @@ func NewCmdSimpleFS(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comm
 	}
 }
 
-// SimpleFs will be expecting slashes, without the mount name
 func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
-	cli, err := GetKBFSMountClient(g)
-	var mountDir string
-	if err == nil {
-		mountDir, _ = cli.GetCurrentMountDir(context.TODO())
-	}
-	if mountDir == "" {
-		mountDir = "/keybase"
-	}
+	mountDir := "/keybase"
```

Comment:
You also have to call some sort of `realpath` on the given path to resolve any symlinks in the current working directory.  I'm skeptical of this strategy in general, as I've noted on the JIRA ticket, because I think we're going to find surprising situations of platforms where it fails.  But let's give it a try and see how it goes.

I did a few small tests on this [go library](https://github.com/yookoala/realpath) and it at least solves the simple case of resolving symlinks in your working directory.  That was only on one platform and one use case.  But we can try it and see how it goes.